### PR TITLE
[wave2-983] moving the FOIP outside the webform

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/js/ucgy_core_variable_table.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    // get the variable table ownership for the form
+    var table_ownership = Drupal.settings.vtownership.form_ownership;
+
+    // loop through each ownership checkbox
+    $( "#edit-ownership .form-type-checkbox input" ).each(function() {
+
+      // if the table ownership string contains the value of this checkbox, check it
+      if (table_ownership.includes($(this).val())) {
+        $(this).attr("checked", "checked");
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/js/ucgy_core_variable_table.js]: Should change this to prop

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc [hunk 2]
```diff
@@ -69,8 +69,8 @@ function ucgy_fgs_field_default_field_bases() {
     'module' => 'list',
     'settings' => array(
       'allowed_values' => array(
-        'Master' => 'Master',
-        'PhD' => 'PhD',
+        'masters' => 'Masters',
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: masters should be singular master

Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: updated.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/js/ucgy_core_variable_table.js (hunk 1 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/js/ucgy_core_variable_table.js]: changed.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/ucgy_core_variable_table_form.module [hunk 3]
```diff
@@ -14,12 +14,35 @@ function ucgy_core_variable_table_form_menu() {
...
+
+  /* don't need this, but might later
+  // if we are checking for more than being logged in
+  if ($user->uid == 1) {
+    return TRUE;
+  }
+  */
+
+  //
+  if (!$user->uid) {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/ucgy_core_variable_table_form.module]: We should probably also include a check for the Support role role here.

**Pull request merged**